app/mailers/wishlist_item_purchase_mailer.rb

```diff
@@ -0,0 +1,9 @@
+class WishlistItemPurchaseMailer < ApplicationMailer
```

Comment:
Mailers should be more generalized than this - you can merge this with the other Wishlist mailers and just call it `WishlistMailer`